mqtt/mqtt-broker/src/config_bad_value_type.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
let's move all files that are for tests to the `mqtt/mqtt-broker/test/` folder

Reply:
moved